src/Compilers/CSharp/Portable/Parser/LanguageParser.cs

```diff
@@ -10359,6 +10363,8 @@ private bool IsAwaitExpression()
                     case SyntaxKind.TrueKeyword:
                     case SyntaxKind.FalseKeyword:
                     case SyntaxKind.StringLiteralToken:
+                    case SyntaxKind.SingleLineRawStringLiteralToken:
+                    case SyntaxKind.MultiLineRawStringLiteralToken:
```

Comment:
Do we have a test affected by this change?

Reply:
i'll add one.

Reply:
I assume this got covered by `TestAwaitRawStringLiteral`.

Reply:
yes.